R/utils.R

```diff
@@ -67,9 +67,20 @@ is_url <- function(x) {
 
 # Given the path to a file, return a file:// URL.
 file_url <- function(filename) {
-  if (is_windows()) {
-    paste0("file://", normalizePath(filename, mustWork = TRUE))
+  if (is_windows() & getRversion() < R_system_version("4.2.0")) {
+    enc2 <- identify
```

Comment:
I think you meant `identity()` here

```suggestion
    enc2 <- identity
```